modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating user workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create a user workloads in Local Zone subnets. 
+
+After the `openshift-installer` creates the cluster, the installer automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod to a node if the pod's deployment does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+
+The `openshift-installer` creates the Machine Set manifest and attaches `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels to each edge worker node that is located in a Local Zone subnet.
+
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You configured an AWS Local Zones environment. 
+* You deployed your cluster om a compatible cloud platform. 
+
+.Procedure
+
+. Create a `deployment` resource YAML file for an edge worker node that operates in a Local Zone subnet. Ensure that you specify the correct tolerations for your edge worker node, so that the scheduler can schedule any pods with matching taints to the cluster.
++
+.Example of a configured portion of the worker manifest file that the VPC uses to run workloads
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment <1>
+metadata:
+  name: <local_zone_name> <2>
+  namespace: <local_zone_namespace> <3>
+spec:
+  selector: 
+    matchLabels:
+      app: <local_zone_application>
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        app: <local_zone_application>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME} <4>
+    spec:
+      nodeSelector:
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME}
+      tolerations: <5>
```

Comment:
@mtulio . Where does one define a taint? For example: https://github.com/openshift/enhancements/pull/1232/files#diff-f2590494c9138551ab5b3d5632d11170fe8742a128256f9e1d8ebc63baa19f41R437-R440

Reply:
the node is created with the taint defined on the MachineSet (linked by you).
This toleration must match with the taints to allow scheduling the pod on the edge worker node.

Reply:
Updated to 

>  Sets the tag where the scheduler can apply a taint to affected nodes. Toleration must match the taint that you defined in the machine set manifest file.